src/devices/acurite.c

```diff
@@ -450,8 +450,8 @@ static int acurite_6045_decode (bitrow_t bb, int browlen) {
     data = data_make(
        "time",			"",			DATA_STRING,	time_str,
        "model",	        	"",			DATA_STRING,	"Acurite Lightning 6045M",
-       "sensor_id",		"",			DATA_INT,	sensor_id,
-       "channel",  		"",     		DATA_STRING, 	&channel_str,
+       "sensor_id",    		NULL,  			DATA_FORMAT,    "0x%02X",   DATA_INT,       sensor_id,
```

Comment:
Note, two issues with this, first, the size is greater than 8 bits.  since these are int values, might as well output them as decimal like the others, instead of a hex string.

Reply:
Ok, understood. Will you change that in the PR you said you'll work on or should I change it?

Reply:
I've got it.